plugins/generator-1.20.4/neoforge-1.20.4/plant.definition.yaml

```diff
@@ -20,6 +20,12 @@ templates:
     condition: generateFeature
     name: "@MODDATAROOT/worldgen/placed_feature/@registryname.json"
 
+  # Loot table
+  - template: json/block_loot_table.json.ftl
+    writer: json
+    condition: "generateLootTable()"

```

Comment:
For boolean-type conditions, " are usually omitted:

![image](https://github.com/MCreator/MCreator/assets/16374228/c1beeb42-26f5-41ae-9450-b41f1373322b)